maze/README.md

```diff
@@ -0,0 +1,26 @@
+# Maze game
+
+This is a maze game where Breadth First Search has been implemented. You now have to implement the Depth First Search algorithm or the A star algorithm.
+
+---
+
+## Instructions
+
+Run the `start.sh` in the maze folder
+ - This will download required dependencies and run the program
+
+```bash
+
+# Set permissions to execute the script
+chmod +x start.sh
```

Comment:
This is fine, but I think git also supports committing the +x bit such that any clone of this repo would already have +x on `start.sh`.